controllers/storagecluster/cephobjectstores.go

```diff
@@ -272,3 +283,12 @@ func getCephObjectStoreGatewayInstances(sc *ocsv1.StorageCluster) int {
 	}
 	return defaults.CephObjectStoreGatewayInstances
 }
+
+func isPortableOSD(sc *ocsv1.StorageCluster) bool {
```

Comment:
The helper func checks if any of the OSDs are portable, So instead of isPortableOSD I would suggest hasPortableOSD, Also as the result of the func is utilized for a -ve check we can optimize this function to be 
hasNonPortable OSD and return immidiately when we see a non-portable OSD

Reply:
done.